monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+[id="configuring-the-monitoring-stack"]
+= Configuring the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: configuring-the-monitoring-stack
+
+toc::[]
+
+The {product-title} 4 installation program provides only a low number of configuration options before installation. Configuring most {product-title} framework components, including the cluster monitoring stack, happens post-installation.
+
+This section explains what configuration is supported, shows how to configure the monitoring stack, and demonstrates several common configuration scenarios.
+
+== Prerequisites
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/scaling-cluster-monitoring-operator.html[Scaling the Cluster Monitoring Operator] and verify that you have sufficient resources.
+
+// Maintenance and support for monitoring
+include::modules/osd-monitoring-maintenance-and-support.adoc[leveloffset=+1]
+include::modules/osd-monitoring-support-considerations.adoc[leveloffset=+2]
+include::modules/osd-monitoring-unmanaged-monitoring-operators.adoc[leveloffset=+2]
+
+// Configuring the monitoring stack
+include::modules/osd-monitoring-configuring-the-monitoring-stack.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
```

Comment:
The "Preparing to configure the monitoring stack" subsection does not exist in this ROSA book. Is this because the steps to create the config map is not relevant to this platform? You could either remove or change this cross-reference to reflect this.

Reply:
I removed the xref as it doesn't apply.